ansible/vars/topo_smartswitch-ha.yml

```diff
@@ -0,0 +1,100 @@
+topology:
+  dut_num: 2
+  VMs:
```

Comment:
Are we missing the DPU KVM here?

Reply:
I can add DPU KVMs, but this was mainly for physically testbeds (DPU KVMs related tasks will be skipped.)